services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -37,16 +40,32 @@ object WorkflowQueryKey {
     override val name = "Name"
 
     override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[List[String]] = {
-
       val values = valuesFromMap(grouped).toList
-      val nels = values map {
+      val nels:List[data.ValidatedNel[String,String]] = values map {
```

Comment:
Intellij now bothers us to annotate public methods and variables. So, unless we can make this private...?